frontend/packages/data-portal/app/state/filterHistory.ts

```diff
@@ -59,10 +49,10 @@ export function useDepositionHistory() {
 
 export function useSyncParamsWithState({
```

Comment:
could you add some documentation on what this function does? All the props need to be stable right, cuz this is only supposed to run once when the route loads?

Reply:
Also a little confused by the name now 😆 cuz it sounds like it's syncing search params with local state, but the prop `setParams` is actually for local state?

Reply:
Oh nvm about the first comment, now I see it's supposed to keep running when filters change, and I guess `searchParams` changes so it all works 👍